x/emissions/migrations/v2/migrate.go

```diff
@@ -292,8 +292,9 @@ func restoreAllRecordCommits(store storetypes.KVStore, cdc codec.BinaryCodec, co
 	return nil
 }
 
-func removeOldKVStores(store storetypes.KVStore) {
-	store.Delete(types.ChurnableTopicsKey)
-	store.Delete(types.TopicLastWorkerPayloadKey)
-	store.Delete(types.TopicLastReputerPayloadKey)
-}
+// staticcheck is saying this is never used, can we delete it??
+// func removeOldKVStores(store storetypes.KVStore) {
+// 	store.Delete(types.ChurnableTopicsKey)
+// 	store.Delete(types.TopicLastWorkerPayloadKey)
+// 	store.Delete(types.TopicLastReputerPayloadKey)
+// }
```

Comment:
@fernandofcampos @guilherme-brandao @RedBird96  ?? probably should be called above

Reply:
this is an old version I guess

Reply:
fixed it